packages/web-components/fast-foundation/CEMToMarkdown.mjs

```diff
@@ -216,7 +216,24 @@ function replaceJSDOCLinksWithMDLinks(text)
         {
             text = text.replace(jsdocLink, "[" + linkParts[1].trim() + "](" + linkParts[0].trim() + ")");
         }
-        startIndex = text.indexOf('{@link',endIndex);
+        startIndex = text.indexOf('{@link');
+    }
+    return text;
+}
+
+function removeCommentBlocks(text)
+{
+    let startIndex = text.indexOf('/*');
+    if(startIndex < 0) return text;
```

Comment:
Wrap the body of if statements in curly braces

Reply:
Done